manifests/8/8GadgetPack/net/8GadgetPack/36.0.0/8GadgetPack.net.8GadgetPack.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 8GadgetPack.net.8GadgetPack

```

```suggestion
PackageIdentifier: 8GadgetPack.8GadgetPack
```
Package folder will need to be updated when identifier is changed